Tests/FunctionalTests/AEPEdgeFunctionalTests.swift

```diff
@@ -389,6 +390,8 @@ class AEPEdgeFunctionalTests: FunctionalTestBase {
         XCTAssertEqual(7, requestBody.count)
         resetTestExpectations()
 
+        sleep(2)
```

Comment:
This is a bit of an edge case in testing. This specific test case requires that we get a response for the first event and update the store payloads, then send out the second event with the store payloads. Everything was synonymous with the original implementation, so this worked fine; however, it is asynchronous with the HitQueue, so we need to add a sleep between the first and second events to give it a chance to process the hit and response.

Reply:
We could, however, that'd increase waits for all other tests that may not need the additional sleep time.

Reply:
I am playing around with this and it doesn't appear to make a difference. I increased the timeout to something high like 20 seconds and it seems to just execute immediately. Going to look into this.

Reply:
Adding a wait time for the network request shouldn't affect this, it shouldn't wait for the entire timeout if the request is executed earlier.